src/helpers.ts

```diff
@@ -80,7 +80,7 @@ export function lintExpressionStatement(
 ) {
   if (!node?.expression?.name) return;
   const failingRule = rules.find(
-    (rule) => rule.object === node.expression.name
+    (rule) => rule.object === node?.expression?.name
```

Comment:
In the case that `node?.expression?.name` and `rule.object` are `undefined`, this will return true. What are your thoughts on something like this:

```js
  const failingRule = node?.expression?.name && rules.find(
    (rule) => rule.object === node.expression.name
```

Reply:
Doesn't L81 ` if (!node?.expression?.name) return;` prevent this?

Reply:
Ah right, my bad